software-containers/README.md

```diff
@@ -16,3 +16,18 @@ group:      sss files
 
 ### xterm
 `xterm` must be installed on the container
+
+### ENTRYPOINT
+Use ENTRYPOINT to specify an executable to be run, for example a startup script which can execute several tasks when the container starts up.
+
+### CMD
+Use CMD to specify a command to be executed at the end of the Dockerfile, "xterm".
```

Comment:
What happens if both ENTRYPOINT and CMD are provided?  What does that parameter mean in each scenario?  These types of questions should be answered.  I think maybe a table or matrix with all the combinations and results might be the best way to capture this.